test/cmocka/src/audio/mux/mock.c

```diff
@@ -16,19 +16,19 @@
 
 static struct sof sof;
 
-struct tr_ctx buffer_tr;
+//struct tr_ctx buffer_tr;
 
 void pipeline_xrun(struct pipeline *p, struct comp_dev *dev, int32_t bytes)
 {
 }
 
-void comp_update_buffer_produce(struct comp_buffer *buffer, uint32_t bytes)
-{
-}
-
-void comp_update_buffer_consume(struct comp_buffer *buffer, uint32_t bytes)
-{
-}
+//void comp_update_buffer_produce(struct comp_buffer *buffer, uint32_t bytes)
+//{
+//}
+//
+//void comp_update_buffer_consume(struct comp_buffer *buffer, uint32_t bytes)
+//{
+//}
```

Comment:
Not sure how kosher this comment-out is.

Reply:
not kosher at all. cleaned up.